demo/app/components/avatar-stack.ts

```diff
@@ -9,8 +9,12 @@ import type { SpaceMember } from '../../../src/Space';
 
 dayjs.extend(relativeTime);
 
-const updateStatusTime = (statusEl: Element, timestamp: number) =>
-  (statusEl.innerHTML = `Left ${dayjs().to(timestamp)}`);
+const updateStatusTime = (statusEl, timestamp) => {
+  const diffInSeconds = dayjs().diff(timestamp, 'second');
+  diffInSeconds === 1
+    ? (statusEl.innerHTML = 'Last seen 1 second ago')
+    : (statusEl.innerHTML = `Last seen ${diffInSeconds} seconds ago`);
```

Comment:
A slightly different way to write this would be:

```js
statusEl.innerHTML = `Last seen ${diff} second${diff > 1 ? 's' : ''} ago`
```

2 questions:
- could diffInSeconds ever be 0?
- does `dayjs().diff(timestamp, 'second')` round to integers?

Reply:
oh your way is so much nicer! thanks :)
and to answer your questions:
* yes
* yes

Reply:
And we can account for 0 seconds like this:
```
statusEl.innerHTML = `Last seen ${diffInSeconds} second${diffInSeconds === 1 ? '' : 's'} ago`;
```